### DISCUSSION TITLE
Simplify route data and action function names.

### DISCUSSION THREAD

**Discussion by izznat** [solidjs/solid-start#315] [Ideas]
Currently there are isomorphic and server only variation of data and action function. Considering both server only data and action function has `$` suffix on its name, I propose below naming changes:

- `createServerData$` -> `createRouteData$`
- `createServerAction$` -> `createRouteAction$`
- `createServerMultiAction$` -> `createRouteMultiAction$`

The `$` sign is enough to be the indicator of them being executed only on the server side. While also make it clear that those functions are the variants of the ones without the `$` sign suffix. I'm interested to know others opinion!

**Comment by ryansolid** [comment]
That's interesting. The $ was added because we felt it wasn't obvious enough these were special but you are right it is sufficient to differentiate. The only question is it too similar to lead to people missing the $ and not realizing it?

**Reply by izznat** [reply]
Yeah, that's a possibility. No way to know for sure without people try using it on daily basis. But, at least for me, the `$` sign stands out more than alphabet characters. So I can't help but notice it.
